server/index.js

```diff
@@ -13,14 +13,23 @@ export default app => {
     }
 
     app.get('/workato-jwt', (req, res) => {
-        const token = sign(
-            {
-                sub: `${process.env.WK_API_KEY}:${process.env.WK_USER_ID}`,
-                jti: nanoid()
-            },
-            process.env.WK_JWT_PRIVATE_KEY,
-            {algorithm: 'RS256'}
-        );
+        const {WK_API_KEY, WK_CUSTOMER_ID, WK_USER_ID, WK_CUSTOM_VENDOR_ORIGIN} = process.env;
+        const subParams = [WK_API_KEY, WK_CUSTOMER_ID];
```

Comment:
Will `WK_CUSTOMER_ID` always defined? Looks like it is new env variable. Maybe you should check it, like you do for  `WK_USER_ID`.

Reply:
yes, WK_CUSTOMER_ID is always defined. basically WK_CUSTOMER_ID is the customer team ID and WK_USER_ID is a valid member within the customer team.

Reply:
Got it.